### PR TITLE
en: update the eks deployment doc

### PR DESCRIPTION
**Pull request by shenli**
### What is changed, added or deleted? (Required)

Translate https://github.com/pingcap/docs-tidb-operator/pull/421

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/421
- Other reference link(s): https://github.com/pingcap/tidb-operator/pull/2714

### CODE REVIEW

File: en/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -86,14 +86,44 @@ git clone --depth=1 https://github.com/pingcap/tidb-operator && \
...
+* Deploy a cluster for production environment
+
+  The production setup creates a new VPC, a t2.micro instance as the bastion host and an EKS cluster with the following Amazon EC2 instances as worker nodes:
+
+  * 3 c5.xlarge instances for PD
+  * 3 i3.4xlarge instances for TiKV
+  * 2 c5.2xlarge instances for TiDB
+  * 1 c5.2xlarge instances for monitor
+
+  To use the production setup, plesae use the following command:
```
Comment (ran-huang) [inline en/deploy-on-aws-eks.md:118]:
```suggestion
  To use the production setup, run the following command:
```

File: en/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -86,14 +86,44 @@ git clone --depth=1 https://github.com/pingcap/tidb-operator && \
...
+
+  {{< copyable "shell-regular" >}}
+
+  ``` shell
+  cp prod.tfvars terraform.tfvars
+  ```
+
+Beside the setups mentioned above, you can modify `terraform.tfvars` to set the value of variables and configure the cluster as needed. See the variables that can be set and their descriptions in `variables.tf`.
```
Comment (ran-huang) [inline en/deploy-on-aws-eks.md:126]:
```suggestion
Besides the setups mentioned above, you can modify `terraform.tfvars` to set the value of variables and configure the cluster as needed. See the configurable variables and their descriptions in `variables.tf`.
```

**Comment by shenli** [comment]
@ran-huang PTAL

File: en/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -86,14 +86,43 @@ git clone --depth=1 https://github.com/pingcap/tidb-operator && \
...
+
+    To use the production setup, run the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    cp prod.tfvars terraform.tfvars
+    ```
+
+Besides the setups mentioned above, you can modify `terraform.tfvars` to set the value of variables and configure the cluster as needed. See the configurable variables and their descriptions in `variables.tf`.
```
Comment (lilin90) [inline en/deploy-on-aws-eks.md:125]:
```suggestion
In addition to the setups mentioned above, you can also modify `terraform.tfvars` to set the value of variables and configure the cluster as needed. See the configurable variables and their descriptions in `variables.tf`.
```

**Pull request merged**